### ISSUE TITLE
Latest FPP V7 Beta updates seem to have broken fpp-after-hours

### ISSUE THREAD

**Issue by majkrebs** [jcrossbdn/fpp-after-hours#20]
![image](https://github.com/jcrossbdn/fpp-after-hours/assets/10644549/3a879d85-30ea-4fa3-a9a1-36bfac20f140)
![image](https://github.com/jcrossbdn/fpp-after-hours/assets/10644549/c510a3be-e38f-4f1e-a890-b0b7e493f55f)

Was working on initial V7 Beta versions but is broken tonight after loading latest updates. Reboot, uninstall/install, and re-installs of "additional software" have not worked to fix it. Point me to a log file or something and I will get you whatever I have that will help.

**Comment by jcrossbdn** [comment]
Please run “apt install mpd mpc” if it won’t installTo see if it’s just a package/distro issue.  If it is installed then fpp-after-hours runs this command to test if it is installed: “mpc status” I believe (will have to confirm tomorrow).I have had a few users with issues installing mpc/mpd through the plug-in over the past few years but usually it sorts out after manually installing (or solving dns issues is the other big issue).My schedule is full the next few weeks but happy to switch to the 7x latest to see what I see after that.  Please let me know if you find anything from this that requires a code change or documentation blurb added.JasonSent from my iPhoneOn Jul 4, 2023, at 10:32 PM, majkrebs ***@***.***> wrote:﻿


Was working on initial V7 Beta versions but is broken tonight after loading latest updates. Reboot, uninstall/install, and re-installs of "additional software" have not worked to fix it. Point me to a log file or something and I will get you whatever I have that will help.

**Comment by majkrebs** [comment]
fpp@FPP01:~ $ sudo apt install mpd mpc
Reading package lists... Done                                                                                                                                         
Building dependency tree... Done                                                                                                                                      
Reading state information... Done                                                                                                                                     
mpc is already the newest version (0.33-1).                                                                                                                           
mpd is already the newest version (0.22.6-1).                                                                                                                         
0 upgraded, 0 newly installed, 0 to remove and 92 not upgraded.                                                                                                       
fpp@FPP01:~ $

fpp@FPP01:~ $ mpc status
MPD error: Connection refused                                                                                                                                         

fpp@FPP01:~ $ mpd
exception: no such user "mpd"                                                                                                                                         

I used to be friends with Linux but it is too many gray hairs ago.

**Comment by majkrebs** [comment]
Practicing my Linux skills I SSH'd to FPP and removed mpc and mpd and all their dependencies. Rebooted and went back in and attempted a install.

dpkg: unrecoverable fatal error, aborting:
 unknown system user 'mpd' in statoverride file; the system user got removed
before the override, which is most probably a packaging bug, to recover you
can remove the override manually with dpkg-statoverride
E: Sub-process /usr/bin/dpkg returned an error code (2)

So, then I learned about "dpkg-statoverride remove /path/problem". dpkg-statoverride --list showed which paths were tied to user mpd. Removed all of them (4).

Did a full reinstall from SSH after a reboot. Had a failure reported near the end of the install but couldn't find an actual problem. So ran it again and it reported all was installed.

Rebooted and have SUCCESS. 
After hours is running again!

Probably can close this as it might be hard to recreate. Also, might suggest to people to run the install from a terminal  instead of by clicking the install script. For me, I got better visibility into the errors the install was seeing which led me to the dpkg-statoverride command.
